src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -358,6 +343,7 @@ internal static unsafe void FixupMethodCell(IntPtr hModule, MethodFixupCell* pCe
             IntPtr pTarget;
 
 #if FEATURE_OBJCMARSHAL
```

Comment:
It looks like this method could easily move to InteropHelpers.macOS.cs too

Reply:
I am not sure how it would look like.

Reply:
It may be possible to move everything from `InteropHelpers.macOS.cs` to `ObjectiveCMarshal.NativeAot.cs` so that `InteropHelpers` is not turning into everything in the kitchen sink kind of type.

Reply:
I moved almost everything into `ObjectiveCMarshal.NativeAot.cs`. This simplifies the things a decent amount. What do you think?